tide_landing_page.install

```diff
@@ -2479,16 +2478,49 @@ function tide_landing_page_update_8046() {
 /**
  * Remove depricated paragraph featured_news.
  */
-function tide_landing_page_update_8047() {
+function tide_landing_page_update_8047(&$sandbox) {
```

Comment:
Looks correct to me - do we need a new update number, has this run already on any sites?

Reply:
nope, this one will go in the next release. Vincent found the issue on a testing PR in content-vic.